chainercv/links/detection_link.py

```diff
@@ -0,0 +1,39 @@
+import chainer
+
+
+class DetectionLink(chainer.Link):
```

Comment:
I used `chainer.Link` because it is most basic class. If you want to have child links, you can write `class MyDetector(chianer.Chain, DetectionLink)`. If you want to treat child links as a list, you can write `class MyDetector(chianer.ChainList, DetectionLink)`.